ui/app/components/pending-tx.js

```diff
@@ -298,7 +310,7 @@ PendingTx.prototype.render = function () {
             type: 'submit',
             value: 'ACCEPT',
             style: { marginLeft: '10px' },
-            disabled: insufficientBalance || !this.state.valid,
+            disabled: insufficientBalance || !this.state.valid || !isValidAddress,
```

Comment:
One of these may be checking if the gas values are correct, while the other one is checking if the address is valid. We may be able to merge the logic for checking isValidAddress into a `setState`